src/style.css

```
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(n),
+.stars--2 :nth-child(n),
+.stars--3 :nth-child(n),
+.stars--4 :nth-child(n),
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star,
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

rewrite only background-image css rule, other properties are already defined for each star above

```suggestion
.stars--0 .stars__star,
.stars--1 :nth-child(n + 2),
.stars--2 :nth-child(n + 3),
.stars--3 :nth-child(n + 4),
.stars--4 :nth-child(n + 5) {
  background-image: url(images/star.svg);
}
```